src/js/options.js

```diff
@@ -53,6 +53,56 @@ async function enableDisableReplaceTab() {
   await browser.storage.local.set({replaceTabEnabled: !!checkbox.checked});
 }
 
+async function backupContainers() {
+  const backupLink = document.getElementById("containers-save-link");
+  const backupResult = document.getElementById("containers-save-result");
+  try {
+    const content = JSON.stringify(
+      await browser.runtime.sendMessage({
+        method: "backupIdentitiesState"
+      })
```

Comment:
Pretty-printing the JSON file would make it a lot easier to read and make diffs, etc.
```js
const content = JSON.stringify(
  await browser.runtime.sendMessage({
    method: "backupIdentitiesState"
  }),
  null,
  2
);
```